src/main/proto/usersGrpc.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.ea.restaurant.grpc";
+
+
+message id{
```

Comment:
Shouldn't you rename this to `FindByIdUserRequest`?

Reply:
Done.